yt/frontends/cholla/tests/test_outputs.py

```diff
@@ -0,0 +1,64 @@
+import yt
+from yt.frontends.cholla.api import ChollaDataset
+from yt.testing import assert_equal, requires_file
+from yt.utilities.answer_testing.framework import data_dir_load
+
+_fields = (
+    ("gas", "temperature"),
+    ("gas", "density"),
+)
+
+cholla_0 = "cholla_0.h5"
```

Comment:
Tests are failing because the file isn't found on server. I think this is the fix
```suggestion
cholla_0 = "ChollaSimple/0.h5"
```